src/db/z_db.erl

```diff
@@ -765,18 +778,20 @@ table_exists(Table, Context) ->
 
 
 %% @doc Make sure that a table is dropped, only when the table exists
+-spec drop_table(table_name(), #context{}) -> ok.
```

Comment:
The spec in line 781 uses a record, please define a type for the record and use that instead.